.travis.yml

```diff
@@ -11,6 +11,29 @@ env:
   - KARTOTHEK_ARROW_VERSION=0.12.1
   - KARTOTHEK_ARROW_VERSION=0.13.0
 
+
+matrix:
+  include:
+  - name: "Integration test for Numpy Pandas"
+    if: branch=numpan_nightly
+    python: "3.7"
+    env:
+      - NUMPY_PANDAS_INTEGRATION=1
+    before_script:
+      - ./ci/before_script.sh
+      - ./ci/set_numpy_pandas_integration.sh
+  - name: "Integration test for PyArrow"
+    if: branch=numpan_nightly
+    python: "3.7"
+    env:
+      - PYARROW_INTEGRATION=1
+    before_script:
+      - ./ci/before_script.sh
+      - ./ci/set_pyarrow_integration.sh
+    script:
+      - python ./tests/integration/pyarrow_integration_test.py
+
+
 before_install:
```

Comment:
`before install` steps can also be moved to a script but I think it's also fine to leave it as-is.

Reply:
ya, for now we will keep this as- is